airline-core/src/main/java/com/github/rvesse/airline/HelpOption.java

```diff
@@ -41,6 +42,7 @@
     private GlobalMetadata<C> globalMetadata;
```

Comment:
Does this not also need `@Nullable`?

If you use this in a `SingleCommand` command there won't be any `GlobalMetadata`

Reply:
Yes, you are right. I've added a test case for a single command with help option.